controllers/topics.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { fetchTopics } = require('../Models/topics.models')
+
+
+exports.getTopics = (req, res) => {
+    fetchTopics().then((data) => res.status(200).send({ data }))
+
+};
+
+
+
+
+
+exports.sendSnacks = (req, res) => {
+    selectSnacks().then((snacks) => res.status(200).send({ snacks }));
+  };
```

Comment:
Don't forget to remove any code that isn't a part of the project/ticket ☝️

Reply:
Oh yeah, sorry Christian was using this as a template from a previous project it is now removed.